recipes-browser/wpewebkit/wpewebkit/Fix-LBSE-blurriness.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/Source/WebCore/platform/graphics/GraphicsLayer.h b/Source/WebCore/platform/graphics/GraphicsLayer.h
```

Comment:
```suggestion
Fix blurry image when with LBSE is on

Upstream-Status: Inappropriate [meta-wpe-image specific]

diff --git a/Source/WebCore/platform/graphics/GraphicsLayer.h b/Source/WebCore/platform/graphics/GraphicsLayer.h
```

Reply:
OK. It is fine. I've added just a small suggestion based on Yocto guides: https://wiki.yoctoproject.org/wiki/Best_Known_Methods_%28BKMs%29_for_Package_Updating#Patch_Comments

Reply:
Done.